### PR TITLE
Number tokenization

### PR DESCRIPTION
**Pull request by sultur**
Modifications to tokenizer which makes parsing written numbers less aggressive.
(Also typo fixes and whitespace fixes in Abbrev.conf)

### CODE REVIEW

**Review by vthorsteinsson — COMMENTED** [review]
We need a further think on how to handle number words. I'm not sure we're on the right track by returning them as Tok.NUMBER. A better strategy might be to return them as Tok.WORD with a special val field, or create a new token kind, Tok.NUMWORD or something like that. This requires changes in GreynirPackage, but would solve a couple of long-standing problems.

File: test/test_tokenizer.py [hunk 1]
```diff
@@ -1714,6 +1864,245 @@ def test_abbrev() -> None:
...
+        Tok(kind=TOK.WORD, txt="níutíu", val=None),
+        Tok(
+            kind=TOK.WORD,
+            txt="þús.kr.",
+            val=[BIN_Tuple("þúsundir króna", 0, "kvk", "skst", "þús.kr.", "-")],
+        ),
+        Tok(kind=TOK.S_END, txt=None, val=None),
+    ]
+
+    tokens = list(t.tokenize("Aðgerðin kostaði 14 þús.kr."))
```
Comment (vthorsteinsson) [inline test/test_tokenizer.py:1917]: @sultur Can you please add a test that puts several spaces between 14 and þús.kr. and checks that the original text of the amount token correctly includes these spaces? Also, as another case, try "14þús.kr." without spaces, and that the original text is correct there also?

Comment (sultur) [reply test/test_tokenizer.py:1917]: Good idea! Added two tests for that.

**Review by vthorsteinsson — COMMENTED** [review]
Looks good to me!

**Pull request merged**